plugins/completion/nvim-cmp/sources/codeium-nvim.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.codeium-nvim;
+in {
+  meta.maintainers = [maintainers.GaetanLepage];
+
+  options.plugins.codeium-nvim =
+    helpers.extraOptionsOptions
+    // {
+      package = helpers.mkPackageOption "codeium.nvim" pkgs.vimPlugins.codeium-nvim;
+
+      configPath =
+        helpers.defaultNullOpts.mkNullable (with types; either str helpers.rawType)
```

Comment:
You should now be able to simply use `mkStr`, it should accept raw types

Reply:
Done !